apps/condo/domains/ticket/schema/TicketOrganizationSetting.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Generated by `createschema ticket.TicketOrganizationSetting 'organization:Relationship:Organization:CASCADE; defaultDeadline?:Integer; paidDeadline?:Integer; emergencyDeadline?:Integer; warrantyDeadline?:Integer;'`
+ */
+
+const { GQLListSchema } = require('@condo/keystone/schema')
+const { historical, versioned, uuided, tracked, softDeleted } = require('@condo/keystone/plugins')
+const { dvAndSender } = require('@condo/domains/common/schema/plugins/dvAndSender')
+const access = require('@condo/domains/ticket/access/TicketOrganizationSetting')
+const { ORGANIZATION_OWNED_FIELD } = require('@condo/domains/organization/schema/fields')
+const { generateTicketDefaultDeadlineFields } = require('@condo/domains/ticket/schema/fields/TicketOrganizationSetting')
+
+
+const TicketOrganizationSetting = new GQLListSchema('TicketOrganizationSetting', {
```

Comment:
You need to add UNIQ CONSTRAINS! It's important

Reply:
organization - uniq, line 20